onionperf/measurement.py

```diff
@@ -300,6 +312,8 @@ def __start_tgen(self, name, tgen_port, socks_port=None, server_urls=None):
         if socks_port is None:
             model.ListenModel(tgen_port="{0}".format(tgen_port)).dump_to_file(tgen_confpath)
             logging.info("TGen server running at 0.0.0.0:{0}".format(tgen_port))
+        elif self.oneshot:
+            model.OneshotModel(tgen_port="{0}".format(tgen_port), tgen_servers=server_urls, socksproxy="127.0.0.1:{0}".format(socks_port)).dump_to_file(tgen_confpath)
```

Comment:
If these arguments are the same, can we instead assign the model to a variable and then call that?